ADAL/src/validation/ADAadAuthorityCache.m

```diff
@@ -317,4 +317,48 @@ - (NSURL *)cacheUrlForAuthority:(NSURL *)authority
     return urlForPreferredHost(authority, record.cacheHost);
 }
 
+- (NSArray<NSURL *> *)cacheAliasesForAuthority:(NSURL *)authority
+{
+    NSMutableArray<NSURL *> *authorities = [NSMutableArray new];
+    
+    __auto_type record = [self checkCache:authority];
+    if (!record)
+    {
+        [authorities addObject:authority];
+        return authorities;
+    }
+    
+    NSArray<NSString *> *aliases = record.aliases;
+    NSString *cacheHost = record.cacheHost;
+    NSString *host = authority.host;
```

Comment:
I remember from previous PR that cached host could have the port. Here you're using just the host without port. Wouldn't it result in possible mismatch for cases, when port is provided, since cacheHost can have an optional port? Correct me if I'm wrong :)

Reply:
:(